components/sensor/atm90e32.rst

```diff
@@ -0,0 +1,139 @@
+        voltage_a:
+          name: "EMON Line Voltage A"
+        current_a:
+          name: "EMON CT1 Current"
+        current_b:
+          name: "EMON CT2 Current"
+        power:
+          name: "EMON Total Active Power CT1-2"
+        frequency:
+          name: "EMON Line Frequency"
+        line_freq: 50Hz
+        gain_pga: 2X
+        gain_volt_a: 47660
+        gain_ct_a: 12577
+        gain_ct_b: 12577
+        update_interval: 60s
+
+    # Example CS 6-channel entry
+    sensor:
+      - platform: atm90e32
+        cs_pin: 5
+        voltage_a:
+          name: "EMON Line Voltage A"
+        current_a:
+          name: "EMON CT1 Current"
+        current_b:
+          name: "EMON CT2 Current"
+        current_c:
+          name: "EMON CT3 Current"
+        power:
+          name: "EMON Total Active Power CT1-3"
+        frequency:
+          name: "EMON Line Frequency"
+        line_freq: 50Hz
+        gain_pga: 2X
+        gain_volt_a: 47660
+        gain_ct_a: 12577
+        gain_ct_b: 12577
+        gain_ct_c: 12577
+        update_interval: 60s
+      - platform: atm90e32
+        cs_pin: 4
+        current_a:
+          name: "EMON CT4 Current"
+        current_b:
+          name: "EMON CT5 Current"
+        current_c:
+          name: "EMON CT6 Current"
+        power:
+          name: "EMON Total Active Power CT4-6"
+        line_freq: 50Hz
+        gain_pga: 2X
+        gain_volt_a: 47660
+        gain_ct_a: 12577
+        gain_ct_b: 12577
+        gain_ct_c: 12577
+        update_interval: 60s
+
+.. note::
+
+    The gain values must be calibrated for each installation, see instructions at https://github.com/CircuitSetup/Split-Single-Phase-Energy-Meter#calibration. 
+
+Configuration variables:
+------------------------
+
+- **cs_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The pin CS is connected to.
+- **voltage_a** (*Optional*): Use the channel A voltage value of the sensor in V (RMS).
+  All options from :ref:`Sensor <config-sensor>`.
+- **voltage_b** (*Optional*): Use the channel B voltage value of the sensor in V (RMS).
+  All options from :ref:`Sensor <config-sensor>`.
+- **voltage_c** (*Optional*): Use the channel . voltage value of the sensor in V (RMS).
+  All options from :ref:`Sensor <config-sensor>`.
+- **current_a** (*Optional*): Use the channel A current value of the sensor in amperes. All options from
+  :ref:`Sensor <config-sensor>`.
+- **current_b** (*Optional*): Use the channel B current value of the sensor in amperes. All options from
+  :ref:`Sensor <config-sensor>`.
+- **current_c** (*Optional*): Use the channel C current value of the sensor in amperes. All options from
+  :ref:`Sensor <config-sensor>`.
+- **power** (*Optional*): Use the combined (active) power value of all current channels of the sensor in watts. All options from
+  :ref:`Sensor <config-sensor>`.
+- **line_freq** (*Required*, string): The AC line frequency of the supply voltage. One of ``50Hz``, ``60Hz``
```

Comment:
```suggestion
- **line_freq** (**Required**, string): The AC line frequency of the supply voltage. One of ``50Hz``, ``60Hz``
```

And move this below cs_pin (because it's a required option)